VersioningScheme.md

```diff
@@ -1,37 +1,52 @@
-## USEEIO Model Versioning Scheme
+# USEEIO Model Versioning Scheme
 
 USEEIO model names follow a scheme.
-These model names are only intended to be model identifiers and not full model descriptions.
-Each model version will have its own model description page.
+These model names are intended to identify a model by version and a set of key characteristics. 
+They are independent of the software or authors producing the model.
+They are not full model descriptions.
 
-The following table shows the parts of a model name.
 
-|Name part | Definition | Format | Default| Part Example|
-|---|---|---|---|---|
-|Region prefix | Primary model region | Two-letter state name for state models| Blank, assumes national model| `GA`|
-|Root name | Main model name |String| USEEIO | `USEEIO`|
-|Major version number | Indicates a major methodological/data update | Integer | NA | `1`|
-|Minor version number | Indicates a minor methodological/data update | Integer after a dot| NA| `.1`|
-|Commodity or industry form| Indicates CommodityxCommodity or IndustryxIndustry | 'i' for IxI form | Blank, assumes default CXC| `i`|
-|Level of Detail | Base level of BEA ([see definitions](https://www.bea.gov/sites/default/files/methodologies/industry_primer.pdf#page=17)) | 's' for summary level, 'c' for sector level | Blank, assumes detailed level| `s`|
-|Satellite table/indicator subset | Names of subset of satellite tables used | hyphen followed by 3-6 digit string with letter in CAPS | Blank, assumes all satellite tables availble for that region are present| `-GHG`
+## Naming pattern
+Model name are composed of parts in a clear 
+The naming pattern is a set sequence of these parts with required separator characters.
+```
+{prefix}{root} v{major}.{minor}.{update}{form}{sectors}-{IOyear}-{subset}
+```
+
+## Name parts
+The following table define the parts of a model name.
 
-### Full Examples
+|Name part | Definition | Format | Default|  Example|
+|---|---|---|---|---|
+| prefix | Primary model region | Two-letter state name for state models| Blank, assumes national model| `GA`|
+|root | Main model name |String| USEEIO | `USEEIO`|
+|major |  Major version number. Advances indicate a major methodological/data update | Integer | NA | `2`|
+|minor | Minor version number. Advances indicate a minor methodological/data update | Integer| NA| `0`|
+|update| Update number. Advances indicates a minor fix or data update| Integer| NA| `1`|
+|form| Indicates CommodityxCommodity or IndustryxIndustry | 'i' for IxI form | Blank, assumes default CXC| `i`|
+|sectors | Base level of BEA ([see definitions](https://www.bea.gov/sites/default/files/methodologies/industry_primer.pdf#page=17) or number of sectors | Characters for a defined level - 's' for summary level, 'c' for sector level; Integers for an arbitrary number | Blank, assumes detailed level| `s` or `75`|
```

Comment:
I like the idea of introducing 'number of sectors' in the `{sectors}` part. How about further using`{sectors}` as an indicator for 'original BEA sectors' and 'disaggregated/aggregated'?

- Using a letter like `s` means using 'original BEA Summary sectors', user will be directed to the BEA link to find out the number of sectors.
- Using a number like `75` means the model sectors are hybridized (i.e. has disagg/agg), we and the users familiar with BEA levels will immediately know it 99.9% must be a Summary model. Users not familiar with BEA levels always can open model config to see detailed info.

Reply:
You are suggesting that we use both, right? So the example would changed to `USEEIO v2.0is75-2016-WAT` ?

Reply:
> You are suggesting that we use both, right? So the example would changed to `USEEIO v2.0is75-2016-WAT` ?

No, I'm suggesting we use either. The example should either be `USEEIO v2.0is-2016-WAT` indicating `nodisagg` or `USEEIO v2.0i75-2016-WAT` indicating `withdisagg`.

Reply:
If we no longer provide built-in `nodisagg` models, which is totally fine, then all built-in models will have `number` instead of `letter` to describe `{sectors}` in their names.

Reply:
OK. Yes, my original intent was the letter _OR_ the number of sectors.